<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QWidget" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QGridLayout" name="left_grid" rowstretch="0,0" columnstretch="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLineEdit" name="dir_entry">
         <property name="maximumSize">
          <size>
           <width>144</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="loaddir_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="folder-open"/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="up_dir_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="go-up"/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListWidget" name="filelist">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::LayoutMode::SinglePass</enum>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="currentRow">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="mid_grid">
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="play_button">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start"/>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QVideoWidget" name="video_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="right_vertical">
       <item>
        <widget class="QTextBrowser" name="tag_display_box">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>400</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="write_tag">
           <property name="text">
            <string>Write tag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clear_tag_button">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="user-trash"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="reload_tags_button">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="emblem-synchronized"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tag_config_button">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="document-properties"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QListWidget" name="author_list">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::ResizeMode::Adjust</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ViewMode::IconMode</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="tag_list">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::ResizeMode::Adjust</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ViewMode::IconMode</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
